This patch, by Todd Pfaff <todd@edge.cis.mcmaster.ca>, enlarges the
maximum filename size supported by samba.  It was originaly produced
for samba-1.9.17.

*** clitar.h-1.9.17     Sat May  4 03:50:43 1996
--- clitar.h    Mon Sep  1 18:59:43 1997
***************
*** 1,6 ****
  
  #define TBLOCK 512
! #define NAMSIZ 100
  union hblock {
    char dummy[TBLOCK];
    struct header {
--- 1,6 ----
  
  #define TBLOCK 512
! #define NAMSIZ 1024
  union hblock {
    char dummy[TBLOCK];
    struct header {
